### ISSUE TITLE
Provide online resources for author of plant names 

### ISSUE THREAD

**Issue by mdoering** [tdwg/prior-standards#17]
The prior standard https://github.com/tdwg/prior-standards/tree/master/authors-of-plant-names is still in wide use, but there is no online resource available. As an open TDWG standard should the list of author names not be public and freely available?

Kew sells a book of the 1996 version: http://shop.kew.org/authors-of-plant-names-11180 but otherwise only offers authors through the IPNI search: http://www.kew.org/data/authors.html

**Comment by peterdesmet** [comment]
Who do you think could answer this question?

**Comment by dkoureas** [comment]
Maintaining a list of Author names is an activity linked by default to the Nomenclators. IPNI does provide a list of standardised (but not with UID) list of authors. 
http://www.ipni.org/ipni/authorsearchpage.do

**Comment by mdoering** [comment]
yes @dkoureas, but being a TDWG standard should that full list not be openly available? Even if it is the frozen version in time when the standard was accepted

**Comment by dkoureas** [comment]
@mdoering I think we need to speak with @nickynicolson on this.

**Comment by abubelinha** [comment]
So what about this?
